### PR TITLE
gdrive: Fix checkver regex

### PR DESCRIPTION
**Pull request by rasa**

### CODE REVIEW

File: bucket/gdrive.json [hunk 1]
```diff
@@ -16,22 +16,22 @@
...
-                    "find": "Windows 64-bit\\s+\\|\\s+([A-Fa-f\\d]{40})"
+                    "find": "Window(?:s)?\\s+64-bit\\s*\\|\\s*([A-Fa-f\\d]{40})"
                 }
             },
             "32bit": {
                 "url": "https://drive.google.com/uc?id=$matchId32&export=download#/gdrive.exe",
                 "hash": {
                     "url": "https://raw.githubusercontent.com/prasmussen/gdrive/master/README.md",
-                    "find": "Window(?:s)? 32-bit\\s+\\|\\s+([A-Fa-f\\d]{40})"
+                    "find": "Window(?:s)?\\s+32-bit\\s*\\|\\s*([A-Fa-f\\d]{40})"
```
Comment (Ash258) [inline bucket/gdrive.json]:
```suggestion
                    "regex": "Window(?:s)?\\s+32-bit\\s*\\|\\s*$sha1"
```

File: bucket/gdrive.json [hunk 2]
```diff
@@ -16,22 +16,22 @@
...
+        "regex": "(?sm)gdrive-windows-386.exe\\].+\\?id=(?<id32>[A-Za-z\\d]+)&\\D*([\\d\\.]+).*gdrive-windows-x64.exe\\].+\\?id=(?<id64>[A-Za-z\\d]+)&"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
                 "url": "https://drive.google.com/uc?id=$matchId64&export=download#/gdrive.exe",
                 "hash": {
                     "url": "https://raw.githubusercontent.com/prasmussen/gdrive/master/README.md",
-                    "find": "Windows 64-bit\\s+\\|\\s+([A-Fa-f\\d]{40})"
+                    "find": "Window(?:s)?\\s+64-bit\\s*\\|\\s*([A-Fa-f\\d]{40})"
```
Comment (Ash258) [inline bucket/gdrive.json]:
```suggestion
                    "regex": "Window(?:s)?\\s+64-bit\\s*\\|\\s*$sha1"
```

File: bucket/gdrive.json [hunk 3]
```diff
@@ -16,22 +16,22 @@
     "bin": "gdrive.exe",
     "checkver": {
         "url": "https://raw.githubusercontent.com/prasmussen/gdrive/master/README.md",
-        "regex": "(?sm)gdrive-windows-386.exe\\].+\\?id=(?<id32>[A-Za-z\\d]{28}).*gdrive-windows-x64.exe\\].+\\?id=(?<id64>[A-Za-z\\d]{28})&export=download\\)\\s+\\|\\s+([\\d\\.]+)\\s+\\|\\s+Windows 64-bit"
+        "regex": "(?sm)gdrive-windows-386.exe\\].+\\?id=(?<id32>[A-Za-z\\d]+)&\\D*([\\d\\.]+).*gdrive-windows-x64.exe\\].+\\?id=(?<id64>[A-Za-z\\d]+)&"
```
Comment (Ash258) [inline bucket/gdrive.json]:
```suggestion
        "regex": "(?sm)gdrive-windows-386.exe\\].+\\?id=(?<id32>[A-Za-z\\d]+)&\\D*([\\d.]+).*gdrive-windows-x64.exe\\].+\\?id=(?<id64>[A-Za-z\\d]+)&"
```

File: bucket/gdrive.json [hunk 4]
```diff
@@ -16,22 +16,22 @@
     "bin": "gdrive.exe",
     "checkver": {
         "url": "https://raw.githubusercontent.com/prasmussen/gdrive/master/README.md",
-        "regex": "(?sm)gdrive-windows-386.exe\\].+\\?id=(?<id32>[A-Za-z\\d]{28}).*gdrive-windows-x64.exe\\].+\\?id=(?<id64>[A-Za-z\\d]{28})&export=download\\)\\s+\\|\\s+([\\d\\.]+)\\s+\\|\\s+Windows 64-bit"
+        "regex": "(?sm)gdrive-windows-386.exe\\].+\\?id=(?<id32>[A-Za-z\\d]+)&\\D*([\\d.]+).*gdrive-windows-x64.exe\\].+\\?id=(?<id64>[A-Za-z\\d]+)&"
```
Comment (niheaven) [inline bucket/gdrive.json]:
```suggestion
        "regex": "(?sm)windows-386.*?id=(?<id32>\\w+).*?([\\d.]+).*?id=(?<id64>\\w+)"
```

File: bucket/gdrive.json [hunk 5]
```diff
@@ -16,22 +16,22 @@
...
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
                 "url": "https://drive.google.com/uc?id=$matchId64&export=download#/gdrive.exe",
                 "hash": {
                     "url": "https://raw.githubusercontent.com/prasmussen/gdrive/master/README.md",
-                    "find": "Windows 64-bit\\s+\\|\\s+([A-Fa-f\\d]{40})"
+                    "regex": "Window(?:s)?\\s+64-bit\\s*\\|\\s*$sha1"
```
Comment (niheaven) [inline bucket/gdrive.json]:
```suggestion
                    "regex": "windows-x64.*?$sha1"
```

File: bucket/gdrive.json [hunk 6]
```diff
@@ -16,22 +16,22 @@
...
                 }
             },
             "32bit": {
                 "url": "https://drive.google.com/uc?id=$matchId32&export=download#/gdrive.exe",
                 "hash": {
                     "url": "https://raw.githubusercontent.com/prasmussen/gdrive/master/README.md",
-                    "find": "Window(?:s)? 32-bit\\s+\\|\\s+([A-Fa-f\\d]{40})"
+                    "regex": "Window(?:s)?\\s+32-bit\\s*\\|\\s*$sha1"
```
Comment (niheaven) [inline bucket/gdrive.json]:
```suggestion
                    "regex": "windows-386.*?$sha1"
```

**Comment by rasa** [comment]
`\\w` didn't work, as dashes are in gdrive IDs. Opted for `[^&]` to future-proof

**Pull request merged**